ssl_certs/generate.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
> testdata... (is a folder to) hold ancillary data needed by the tests

[doc](https://pkg.go.dev/cmd/go/internal/test)

I think it is better to use this structure.

Reply:
Yes, thank you!